advanced/Entity Matching with SDK-experimental demo.ipynb

```diff
@@ -406,14 +541,117 @@
     "# Predict on training data\n",
     "job = model.predict(num_matches = 2)\n",
     "matches = job.result\n",
-    "matches[\"items\"] "
+    "pp.pprint((matches[\"items\"]))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note: The model gives a higher score to the correct matches."
+    "#### When to use `feature_type=Bigram-Extra-Tokenizers`?\n",
+    "Similar to bigram, but able to learn that leading zeros and spaces should be ignored in matching. For example ABCDE and 000A<u>B</u>B<u>C</u><u>D</u><u>E</u>1<u>F</u>. Please note that this feature type could give score above 0.5 for two entity strings which are not match at all. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sources = [\n",
+    "    {\"id\":0, \"name\" : \"KKL_21AA1019CA.PV\", \"description\": \"correct\"}, \n",
+    "    {\"id\":1, \"name\" : \"KKL_13FV1234BU.VW\", \"description\": \"ok\"}\n",
+    "]\n",
+    "targets = [\n",
+    "    {\"id\":0, \"name\" : \"000021AA1019CA\", \"description\": \"correct\"}, \n",
+    "    {\"id\":10, \"name\" : \"21KKL1019CA\", \"description\": \"correct\"},\n",
+    "    {\"id\":1, \"name\" : \"21AA1119CA\", \"description\": \"wrong\"}, \n",
+    "    {\"id\":2, \"name\" : \"000013FV1234BU\"},\n",
+    "    {\"id\":3, \"name\" : \"13FV1334BU\", \"description\": \"ok\"},\n",
+    "    {\"id\":13, \"name\" : \"13KKL1234BU\", \"description\": \"ok\"}\n",
+    "]\n",
+    "\n",
+    "model = client.entity_matching.fit(sources = sources,\n",
+    "                                   targets = targets,\n",
+    "                                   match_fields = [(\"name\", \"name\")],\n",
+    "                                   feature_type = \"bigram-extra-tokenizers\"\n",
```

Comment:
Maybe also show what happens if you. use a different feature-type in this situation?